README.md

```diff
@@ -165,3 +166,22 @@ make docker-shell-pgstac # or docker-shell-sqlalchemy
 $ pip install -e stac_fastapi/pgstac[dev]
 $ pytest -v stac_fastapi/pgstac/tests/api/test_api.py 
 ```
+
+### PGAdmin
+
+If you want to use PGAdmin in the same docker compose deployment, just add the following lines to the docker-compose file, this will provide PGAdmin on port 8081:
+
+```yaml
+  pgadmin:
+    image: dpage/pgadmin4
+    volumes:
+      - .:/var/lib/pgadmin/storage/username_email.com/
+    environment:
+      - PGADMIN_DEFAULT_EMAIL=username@email.com
+      - PGADMIN_DEFAULT_PASSWORD=password
+    ports:
+      - "8081:80"
+    command: postgres -N 500
+    links:
+      - "pgstac"
+```
```

Comment:
Thanks! I think the text looks good ... but feels a little specific for the `README.md`. Can you add it to the `tips-and-tricks` page in the documentation? https://github.com/stac-utils/stac-fastapi/blob/main/docs/tips-and-tricks.md